Dockerfile

```
@@ -14,6 +14,13 @@ RUN apt-get update && \
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+# Prep a module to make sure we have all of
+# the required dependencies.
+run pdk new module docker --skip-interview && \
```

```suggestion
RUN pdk new module docker --skip-interview && \
```
Should this be "RUN"?

---

yep!